packages/steveo/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "steveo",
+  "name": "steveo-steveo",
```

Comment:
should not rename this as it will affect publishing

Reply:
i didn't rename it, i just got it working as it was broken 
afaik the module name is specified by the name in package.json, and for the steveo project that is steveo-steveo

i may have missed a trick with turbo

Reply:
.... i specifically didn't change the package.json for steveo as i knew it would affect publishing

Reply:
okay, let me look into that, i may have made a mistake in my turbo learning